<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-47P2MTB" failures="16" tests="57" name="Surefire test" time="96.471" errors="0" timestamp="2024-11-30T19:29:41 IST">
  <testcase classname="api.test.PetTest" name="testAddNewPetPostResponse" time="5.067">
    <failure type="java.lang.AssertionError" message="Response time is outside the expected range (200-3000ms). expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Response time is outside the expected range (200-3000ms). expected [true] but found [false]
at api.test.PetTest.testAddNewPetPostResponse(PetTest.java:119)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testAddNewPetPostResponse -->
  <testcase classname="api.test.PetTest" name="testAddNewPetPostResponse" time="2.087"/>
  <testcase classname="api.test.PetTest" name="testAddNewPetPostResponse" time="1.517"/>
  <testcase classname="api.test.PetTest" name="testUpdateExistingPetPutResponse" time="1.469"/>
  <testcase classname="api.test.PetTest" name="testUpdateExistingPetPutResponse" time="1.385"/>
  <testcase classname="api.test.PetTest" name="testUpdateExistingPetPutResponse" time="1.405"/>
  <testcase classname="api.test.PetTest" name="testFindPetByStatusGetResponse" time="1.521">
    <failure type="org.json.JSONException" message="JSONObject[&amp;quot;name&amp;quot;] not found.">
      <![CDATA[org.json.JSONException: JSONObject["name"] not found.
at org.json.JSONObject.get(JSONObject.java:596)
at api.test.PetTest.testFindPetByStatusGetResponse(PetTest.java:285)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testFindPetByStatusGetResponse -->
  <testcase classname="api.test.PetTest" name="testFindPetByPetIDGetResponse" time="3.473"/>
  <testcase classname="api.test.PetTest" name="testFindPetByPetIDGetResponse" time="1.531"/>
  <testcase classname="api.test.PetTest" name="testFindPetByPetIDGetResponse" time="4.194">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at api.test.PetTest.testFindPetByPetIDGetResponse(PetTest.java:390)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testFindPetByPetIDGetResponse -->
  <testcase classname="api.test.PetTest" name="testUpdatePetPostResponse" time="2.91">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;Object.toString()&amp;quot; because the return value of &amp;quot;io.restassured.response.ExtractableResponse.path(String, String[])&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "Object.toString()" because the return value of "io.restassured.response.ExtractableResponse.path(String, String[])" is null
at api.test.PetTest.testUpdatePetPostResponse(PetTest.java:430)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testUpdatePetPostResponse -->
  <testcase classname="api.test.PetTest" name="testUploadPetImagePostResponse" time="15.663">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at api.test.PetTest.testUploadPetImagePostResponse(PetTest.java:498)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testUploadPetImagePostResponse -->
  <testcase classname="api.test.PetTest" name="testDeletePetDeleteResponse" time="6.478">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at api.test.PetTest.testDeletePetDeleteResponse(PetTest.java:556)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testDeletePetDeleteResponse -->
  <testcase classname="api.test.PetTest" name="deleberatlyFailTest" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at api.test.PetTest.deleberatlyFailTest(PetTest.java:578)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- deleberatlyFailTest -->
  <testcase classname="api.test.PetTest" name="deleberatlySkippedTest1" time="0.0">
    <skipped/>
  </testcase> <!-- deleberatlySkippedTest1 -->
  <testcase classname="api.test.PetTest" name="deleberatlySkippedTest2" time="0.0">
    <skipped/>
  </testcase> <!-- deleberatlySkippedTest2 -->
  <testcase classname="api.test.StoreTest" name="testPostStoreResponse" time="6.661"/>
  <testcase classname="api.test.StoreTest" name="testGetStoreOrderResponse" time="6.417">
    <failure type="java.lang.AssertionError" message="Response time is outside the expected range (200-2000ms). expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Response time is outside the expected range (200-2000ms). expected [true] but found [false]
at api.test.StoreTest.testGetStoreOrderResponse(StoreTest.java:177)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testGetStoreOrderResponse -->
  <testcase classname="api.test.StoreTest" name="testGetStoreInventoriesResponse" time="1.953">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Response body doesn&amp;apos;t match expectation.
Expected: The content to match the given JSON schema.
error: object instance has properties which are not allowed by the schema: [&amp;quot;array&amp;quot;,&amp;quot;capacitor&amp;quot;,&amp;quot;driver&amp;quot;,&amp;quot;feed&amp;quot;,&amp;quot;hard drive&amp;quot;,&amp;quot;not available&amp;quot;,&amp;quot;pending&amp;quot;,&amp;quot;program&amp;quot;,&amp;quot;send&amp;quot;,&amp;quot;transmitter&amp;quot;,&amp;quot;unavailable&amp;quot;]
    level: &amp;quot;error&amp;quot;
    schema: {&amp;quot;loadingURI&amp;quot;:&amp;quot;#&amp;quot;,&amp;quot;pointer&amp;quot;:&amp;quot;/definitions/Welcome8&amp;quot;}
    instance: {&amp;quot;pointer&amp;quot;:&amp;quot;&amp;quot;}
    domain: &amp;quot;validation&amp;quot;
    keyword: &amp;quot;additionalProperties&amp;quot;
    unwanted: [&amp;quot;array&amp;quot;,&amp;quot;capacitor&amp;quot;,&amp;quot;driver&amp;quot;,&amp;quot;feed&amp;quot;,&amp;quot;hard drive&amp;quot;,&amp;quot;not available&amp;quot;,&amp;quot;pending&amp;quot;,&amp;quot;program&amp;quot;,&amp;quot;send&amp;quot;,&amp;quot;transmitter&amp;quot;,&amp;quot;unavailable&amp;quot;]
error: object has missing required properties ([&amp;quot;placed&amp;quot;,&amp;quot;ratvelleg&amp;quot;,&amp;quot;status&amp;quot;])
    level: &amp;quot;error&amp;quot;
    schema: {&amp;quot;loadingURI&amp;quot;:&amp;quot;#&amp;quot;,&amp;quot;pointer&amp;quot;:&amp;quot;/definitions/Welcome8&amp;quot;}
    instance: {&amp;quot;pointer&amp;quot;:&amp;quot;&amp;quot;}
    domain: &amp;quot;validation&amp;quot;
    keyword: &amp;quot;required&amp;quot;
    required: [&amp;quot;available&amp;quot;,&amp;quot;placed&amp;quot;,&amp;quot;ratvelleg&amp;quot;,&amp;quot;sold&amp;quot;,&amp;quot;status&amp;quot;,&amp;quot;string&amp;quot;]
    missing: [&amp;quot;placed&amp;quot;,&amp;quot;ratvelleg&amp;quot;,&amp;quot;status&amp;quot;]

  Actual: {&amp;quot;sold&amp;quot;:7,&amp;quot;string&amp;quot;:733,&amp;quot;unavailable&amp;quot;:2,&amp;quot;pending&amp;quot;:3,&amp;quot;available&amp;quot;:195,&amp;quot;program&amp;quot;:1,&amp;quot;capacitor&amp;quot;:3,&amp;quot;feed&amp;quot;:2,&amp;quot;driver&amp;quot;:1,&amp;quot;array&amp;quot;:1,&amp;quot;not available&amp;quot;:1,&amp;quot;send&amp;quot;:4,&amp;quot;transmitter&amp;quot;:1,&amp;quot;hard drive&amp;quot;:1}
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Response body doesn't match expectation.
Expected: The content to match the given JSON schema.
error: object instance has properties which are not allowed by the schema: ["array","capacitor","driver","feed","hard drive","not available","pending","program","send","transmitter","unavailable"]
    level: "error"
    schema: {"loadingURI":"#","pointer":"/definitions/Welcome8"}
    instance: {"pointer":""}
    domain: "validation"
    keyword: "additionalProperties"
    unwanted: ["array","capacitor","driver","feed","hard drive","not available","pending","program","send","transmitter","unavailable"]
error: object has missing required properties (["placed","ratvelleg","status"])
    level: "error"
    schema: {"loadingURI":"#","pointer":"/definitions/Welcome8"}
    instance: {"pointer":""}
    domain: "validation"
    keyword: "required"
    required: ["available","placed","ratvelleg","sold","status","string"]
    missing: ["placed","ratvelleg","status"]

  Actual: {"sold":7,"string":733,"unavailable":2,"pending":3,"available":195,"program":1,"capacitor":3,"feed":2,"driver":1,"array":1,"not available":1,"send":4,"transmitter":1,"hard drive":1}

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:107)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:238)
at api.test.StoreTest.testGetStoreInventoriesResponse(StoreTest.java:278)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testGetStoreInventoriesResponse -->
  <testcase classname="api.test.StoreTest" name="testDeleteStoreOrderResponse" time="2.85">
    <failure type="java.lang.AssertionError" message="Expected status code is 200. expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Expected status code is 200. expected [200] but found [404]
at api.test.StoreTest.testDeleteStoreOrderResponse(StoreTest.java:365)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteStoreOrderResponse -->
  <testcase classname="api.test.UserTest" name="createUser_shouldReturn200StatusCode" time="1.481"/>
  <testcase classname="api.test.UserTest" name="createUser_shouldRespondWithinExpectedTime" time="0.002"/>
  <testcase classname="api.test.UserTest" name="createUser_shouldReturnCorrectContentTypeHeader" time="0.002"/>
  <testcase classname="api.test.UserTest" name="createUser_shouldReturnValidResponseFieldTypes" time="0.001"/>
  <testcase classname="api.test.UserTest" name="createUser_shouldReturnCorrectFieldValues" time="0.059"/>
  <testcase classname="api.test.UserTest" name="createUser_shouldMatchJsonSchema" time="0.084"/>
  <testcase classname="api.test.UserTest" name="getUser_shouldReturn200StatusCode" time="1.988"/>
  <testcase classname="api.test.UserTest" name="getUser_shouldRespondWithinExpectedTime" time="0.001">
    <failure type="java.lang.AssertionError" message="Response time is outside the expected range. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Response time is outside the expected range. expected [true] but found [false]
at api.test.UserTest.getUser_shouldRespondWithinExpectedTime(UserTest.java:367)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getUser_shouldRespondWithinExpectedTime -->
  <testcase classname="api.test.UserTest" name="getUser_shouldReturnCorrectContentTypeHeader" time="0.002"/>
  <testcase classname="api.test.UserTest" name="getUser_shouldReturnValidResponseFieldTypes" time="0.002"/>
  <testcase classname="api.test.UserTest" name="getUser_shouldReturnCorrectFieldValues" time="0.132"/>
  <testcase classname="api.test.UserTest" name="updateUser_shouldReturn200StatusCode" time="6.471"/>
  <testcase classname="api.test.UserTest" name="updateUser_shouldRespondWithinExpectedTime" time="0.002">
    <failure type="java.lang.AssertionError" message="Response time is outside the expected range (200-2000ms). expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Response time is outside the expected range (200-2000ms). expected [true] but found [false]
at api.test.UserTest.updateUser_shouldRespondWithinExpectedTime(UserTest.java:593)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- updateUser_shouldRespondWithinExpectedTime -->
  <testcase classname="api.test.UserTest" name="updateUser_shouldReturnCorrectContentTypeHeader" time="0.001"/>
  <testcase classname="api.test.UserTest" name="updateUser_shouldReturnValidResponseFieldTypes" time="0.002">
    <failure type="org.json.JSONException" message="JSONObject[&amp;quot;id&amp;quot;] not found.">
      <![CDATA[org.json.JSONException: JSONObject["id"] not found.
at org.json.JSONObject.get(JSONObject.java:596)
at api.test.UserTest.updateUser_shouldReturnValidResponseFieldTypes(UserTest.java:655)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- updateUser_shouldReturnValidResponseFieldTypes -->
  <testcase classname="api.test.UserTest" name="updateUser_shouldReturnCorrectFieldValues" time="0.095">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;io.restassured.response.ExtractableResponse.path(String, String[])&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "io.restassured.response.ExtractableResponse.path(String, String[])" is null
at api.test.UserTest.updateUser_shouldReturnCorrectFieldValues(UserTest.java:694)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- updateUser_shouldReturnCorrectFieldValues -->
  <testcase classname="api.test.UserTest" name="updateUser_shouldMatchJsonSchema" time="0.07">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Response body doesn&amp;apos;t match expectation.
Expected: The content to match the given JSON schema.
error: object instance has properties which are not allowed by the schema: [&amp;quot;code&amp;quot;,&amp;quot;message&amp;quot;,&amp;quot;type&amp;quot;]
    level: &amp;quot;error&amp;quot;
    schema: {&amp;quot;loadingURI&amp;quot;:&amp;quot;#&amp;quot;,&amp;quot;pointer&amp;quot;:&amp;quot;/definitions/Welcome2&amp;quot;}
    instance: {&amp;quot;pointer&amp;quot;:&amp;quot;&amp;quot;}
    domain: &amp;quot;validation&amp;quot;
    keyword: &amp;quot;additionalProperties&amp;quot;
    unwanted: [&amp;quot;code&amp;quot;,&amp;quot;message&amp;quot;,&amp;quot;type&amp;quot;]
error: object has missing required properties ([&amp;quot;email&amp;quot;,&amp;quot;firstName&amp;quot;,&amp;quot;id&amp;quot;,&amp;quot;lastName&amp;quot;,&amp;quot;password&amp;quot;,&amp;quot;phone&amp;quot;,&amp;quot;userStatus&amp;quot;,&amp;quot;username&amp;quot;])
    level: &amp;quot;error&amp;quot;
    schema: {&amp;quot;loadingURI&amp;quot;:&amp;quot;#&amp;quot;,&amp;quot;pointer&amp;quot;:&amp;quot;/definitions/Welcome2&amp;quot;}
    instance: {&amp;quot;pointer&amp;quot;:&amp;quot;&amp;quot;}
    domain: &amp;quot;validation&amp;quot;
    keyword: &amp;quot;required&amp;quot;
    required: [&amp;quot;email&amp;quot;,&amp;quot;firstName&amp;quot;,&amp;quot;id&amp;quot;,&amp;quot;lastName&amp;quot;,&amp;quot;password&amp;quot;,&amp;quot;phone&amp;quot;,&amp;quot;userStatus&amp;quot;,&amp;quot;username&amp;quot;]
    missing: [&amp;quot;email&amp;quot;,&amp;quot;firstName&amp;quot;,&amp;quot;id&amp;quot;,&amp;quot;lastName&amp;quot;,&amp;quot;password&amp;quot;,&amp;quot;phone&amp;quot;,&amp;quot;userStatus&amp;quot;,&amp;quot;username&amp;quot;]

  Actual: {&amp;quot;code&amp;quot;:200,&amp;quot;type&amp;quot;:&amp;quot;unknown&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;1777178337&amp;quot;}
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Response body doesn't match expectation.
Expected: The content to match the given JSON schema.
error: object instance has properties which are not allowed by the schema: ["code","message","type"]
    level: "error"
    schema: {"loadingURI":"#","pointer":"/definitions/Welcome2"}
    instance: {"pointer":""}
    domain: "validation"
    keyword: "additionalProperties"
    unwanted: ["code","message","type"]
error: object has missing required properties (["email","firstName","id","lastName","password","phone","userStatus","username"])
    level: "error"
    schema: {"loadingURI":"#","pointer":"/definitions/Welcome2"}
    instance: {"pointer":""}
    domain: "validation"
    keyword: "required"
    required: ["email","firstName","id","lastName","password","phone","userStatus","username"]
    missing: ["email","firstName","id","lastName","password","phone","userStatus","username"]

  Actual: {"code":200,"type":"unknown","message":"1777178337"}

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:107)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:238)
at api.test.UserTest.updateUser_shouldMatchJsonSchema(UserTest.java:733)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- updateUser_shouldMatchJsonSchema -->
  <testcase classname="api.test.UserTest" name="updateUser_withInvalidUsername_shouldHandleGracefully" time="1.501"/>
  <testcase classname="api.test.UserTest" name="loginUser_shouldReturn200StatusCode" time="1.412"/>
  <testcase classname="api.test.UserTest" name="loginUser_shouldRespondWithinExpectedTime" time="0.002"/>
  <testcase classname="api.test.UserTest" name="loginUser_shouldReturnCorrectContentTypeHeader" time="0.011"/>
  <testcase classname="api.test.UserTest" name="loginUser_shouldReturnValidResponseFieldTypes" time="0.002"/>
  <testcase classname="api.test.UserTest" name="loginUser_shouldReturnCorrectFieldValues" time="0.045"/>
  <testcase classname="api.test.UserTest" name="loginUser_shouldMatchJsonSchema" time="0.062"/>
  <testcase classname="api.test.UserTest" name="logoutUser_shouldReturn200StatusCode" time="1.504"/>
  <testcase classname="api.test.UserTest" name="logoutUser_shouldRespondWithinExpectedTime" time="0.002"/>
  <testcase classname="api.test.UserTest" name="logoutUser_shouldReturnCorrectContentTypeHeader" time="0.002"/>
  <testcase classname="api.test.UserTest" name="logoutUser_shouldReturnValidResponseFieldTypes" time="0.003"/>
  <testcase classname="api.test.UserTest" name="logoutUser_shouldReturnCorrectFieldValues" time="0.033"/>
  <testcase classname="api.test.UserTest" name="logoutUser_shouldMatchJsonSchema" time="0.065"/>
  <testcase classname="api.test.UserTest" name="deleteUser_shouldReturn200StatusCode" time="1.466"/>
  <testcase classname="api.test.UserTest" name="deleteUser_shouldRespondWithinExpectedTime" time="0.002"/>
  <testcase classname="api.test.UserTest" name="deleteUser_shouldReturnValidResponseFieldTypes" time="0.002"/>
  <testcase classname="api.test.UserTest" name="deleteUser_shouldReturnCorrectFieldValues" time="0.031"/>
  <testcase classname="api.test.UserTest" name="deleteUser_shouldMatchJsonSchema" time="0.069"/>
  <testcase classname="api.test.UserTest" name="deletedUser_shouldNotBeFetched_shouldReturn404Error" time="4.519">
    <failure type="java.lang.AssertionError" message="Response time is outside the expected range (200-2000ms). expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Response time is outside the expected range (200-2000ms). expected [true] but found [false]
at api.test.UserTest.deletedUser_shouldNotBeFetched_shouldReturn404Error(UserTest.java:1376)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- deletedUser_shouldNotBeFetched_shouldReturn404Error -->
  <testcase classname="api.test.UserTest" name="createUsersInBulk_shouldReturn200StatusCode" time="1.506"/>
</testsuite> <!-- Surefire test -->
